jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/log/RuntimeLogPresenter.java

```diff
@@ -53,60 +65,117 @@ public IsWidget getWidget() {
         return view;
     }
 
+    public void setCurrentProcessInstanceId(Long currentProcessInstanceId) {
```

Comment:
Since you are re-factoring the whole thing I'm thinking of removing this ```current``` part from the field names, as it seems redundant.

Reply:
do we even need these setter methods? In theory, the only place where the process instance data should change is when receiving an ProcessInstanceSelectionEvent.